### PR TITLE
docs/guide-ja updated [ci skip]

### PR DESCRIPTION
**Pull request by softark**

### CODE REVIEW

File: docs/guide-ja/layouts-and-widgets.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+レイアウトとウィジェット
+========================
+
+## メイン・レイアウト
+
+`views/layout/layout.twig` file to replace `views/layout/main.php` を置き換える `views/layout/layout.twig` ファイルの例を示します。
```
Comment (SilverFire) [inline docs/guide-ja/layouts-and-widgets.md]: `file to replace` should this be also translated?

File: docs/guide-ja/layouts-and-widgets.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+{
+    // ..
+    public $layout = 'main.twig';
+    // ..
+```
+
+
+`views/layout/main.twig` の中身は次のようになります。
+
+```
```
Comment (SilverFire) [inline docs/guide-ja/layouts-and-widgets.md]:
```
```twig
```

File: docs/guide-ja/layouts-and-widgets.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            //..
+        ],
+        'uses' => ['yii\bootstrap'],
+    ],
+],
+```
+
+以下が login/logout の動的なバリエーションを持つ `navigation` バーのコードです。
+
+```
```
Comment (SilverFire) [inline docs/guide-ja/layouts-and-widgets.md]:
```
```twig
```

File: docs/guide-ja/layouts-and-widgets.md (hunk 1 above)
Comment (softark) [inline docs/guide-ja/layouts-and-widgets.md]: Oh, I forgot to remove some original words that are already translated.

File: docs/guide-ja/layouts-and-widgets.md (hunk 2 above)
Comment (softark) [inline docs/guide-ja/layouts-and-widgets.md]: OK, I got it. I will add it.
But it's also missing in the original do.

File: docs/guide-ja/layouts-and-widgets.md (hunk 3 above)
Comment (softark) [inline docs/guide-ja/layouts-and-widgets.md]: Added 'twig'

**Comment by samdark** [comment]
Thanks!

**Pull request merged**